script/bootstrap_development.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+gem install bundler
+bundle check || bundle install --deployment --path /vendor/bundle --jobs 2 --retry 2
+rake db:create
+rake db:reset
```

Comment:
This is true. Maybe we should skip the database setup and let the developer/user do this manually. What do you think?